contributors/devel/sig-node/separate-mount-namespace.md

```diff
@@ -0,0 +1,212 @@
+# Managing a Separate Mount Namespace for Kubernetes
+
+## What does this mean?
+
+Originally, Kubelet and the Container Runtime all ran in the same mount namespace as
+the host OS (systemd, login shells, system daemons).
+
+![Fig 1: Original mount propagation](images/original-k8s-mount-propagation.png)
+
+_Fig 1: Original mount propagation_
+
+However, in order to support the full range of cloud-native applications, only
+two guarantees need to be made about the relationship of mountpoints:
+
+1. Mounts originating in the host OS are visible to the Container Runtime, OCI
+   hooks, Kubelet, and container volumeMounts with `MountPropagation:
+   HostToContainer` (or `Bidirectional`)
+2. Mounts originating in the Container Runtime, OCI hooks, Kubelet, and
+   container volumeMounts with `mountPropagation: Bidirectional` are visible to
+   each other and container volumeMounts with `MountPropagation:
+   HostToContainer`
+
+The fact that mounts originating in the Container Runtime, OCI hooks, Kubelet,
+and container volumeMounts with `mountPropagation: Bidirectional` were visible
+to the host OS in the original implementations of Kubernetes was accidental to
+the implementation, and there are a number of real-world benefits to changing
+the way Kubernetes is installed in order to hide the container-specific mount
+points from the host OS.
+
+![Fig 2: Container-Runtime-scoped mount propagation](images/hidden-k8s-mount-propagation.png)
+
+_Fig 2: Container-Runtime-scoped mount propagation_
+
+## Benefits
+
+### Systemd Efficiency
+
+Today systemd's CPU usage is adversely affected by a large number of
+mountpoints, especially versions released prior to December 2020. This feature
```

Comment:
Is there a specific semantic version we could include here?

Reply:
Good idea.

Reply:
Found it.  `systemd-248`  Updated with actual release version.